frontend/src/metabase/admin/permissions/components/PermissionsPageLayout/PermissionsEditBar.jsx

```diff
@@ -1,8 +1,9 @@
 import cx from "classnames";
 import PropTypes from "prop-types";
+import { useState } from "react";
 import { t } from "ttag";
 
-import Confirm from "metabase/components/Confirm";
+import { ConfirmationModal } from "metabase/components/ConfirmationModal";
 import EditBar from "metabase/components/EditBar";
 import Button from "metabase/core/components/Button";
```

Comment:
I left the deprecated legacy button because there's some styling that is being applied to the button to match the purple PermissionsBar and I was having trouble getting the same styling to work with the new buttons